cl.conf

```diff
@@ -91,9 +91,24 @@ storage:
       contents:
         inline: |
           #!/bin/bash -e
-          echo "Running install bricks"
-          sleep 2
-          echo "Done install bricks"
+
+          DOCKER_VERSION=$(docker version -f {{.Client.Version}})
+          echo "Docker client version to install ${DOCKER_VERSION}"
+
+          echo "Cleaning up any existing directories"
+          USER_HOME=$(echo /Users/* | head -1)
+          rm -rf ${USER_HOME}/.bricks/bin
+          mkdir -p ${USER_HOME}/.bricks/bin
```

Comment:
What we should probably do here instead of removing the directory is replace it once we've downloaded everything so that there's no chance for ~/.bricks/bin not to exist.

Also, we may have to do the chmod against a directory on the local system. I had some issues trying to change permissions on files on the host from the guest VM. That's part of why the install-minikube is more complicated than it seems it should be.

Reply:
updating